docs/ja/installation.md

```diff
@@ -0,0 +1,39 @@
+# インストール
+
+## 直接ダウンロード
+
+[dist フォルダ](https://github.com/vuejs/vue-validator/tree/dev/dist)を参照してください。dist フォルダにあるファイルは常に最新の安定版であることに注意してください。 - `dev` ブランチのソースは最新ではありません。
+
+## CDN
+
+### jsdelivr
+
+```html
+<script src="https://cdn.jsdelivr.net/vue.validator/2.1.1/vue-validator.min.js"></script>
+```
+
+## NPM
+
+### 安定版
+
+    $ npm install vue-validator
+
+### 開発版
+
+    $ git clone https://github.com/vuejs/vue-validator.git node_modules/vue-validator
+    $ cd node_modules/vue-validator
+    $ npm install
+    $ npm run build
+
+CommonJS で使用されるとき、明示的に `Vue.use()` 経由でバリデーターをインストールしなければなりません:
+
+> :warning: もし `vue-router` を使用している場合は、 `router#start` の前に `Vue.use()` でインストールしなければなりません。
```

Comment:
Thanks for the comment. I installed the gitbook plugins as follows.

```sh
$ cd vue-validator/docs
$ gitbook install
$ gitbook build
```

Run to check on localhost.
```sh
$ gitbook serve
```

`:warning:` was converted into image tag. But I couldn't still see it.
![image](https://cloud.githubusercontent.com/assets/1726225/15043894/dec9f876-130b-11e6-865e-5ce85b1adfd2.png)

The advanced-emoji plugin seems installed successlly on my PC. But the file path on image tag seems different. hmm.
```sh
$ ll  _book/gitbook/plugins/gitbook-plugin-advanced-emoji/emojis/ | grep warning
-rw-r--r--  1 hypermkt  staff  3173  5  5 21:52 warning.png
```